packages/react/src/components/Avatar/Avatar.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import cat1 from '@assets/img/cats/Cat 3.jpg';
+import type { Meta, StoryFn } from '@storybook/react';
+
+import { BriefcaseIcon } from '@navikt/aksel-icons';
+import { Avatar } from '.';
+import { Badge, DropdownMenu } from '../';
+
+type Story = StoryFn<typeof Avatar>;
+
+const meta: Meta = {
+  title: 'Komponenter/Avatar',
+  component: Avatar,
+  parameters: {
+    layout: 'padded',
+  },
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          display: 'flex',
+          gap: 'var(--ds-spacing-2)',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <Story />
+      </div>
+    ),
+  ],
+};
+
+export default meta;
+
+export const Preview: Story = (args) => <Avatar {...args} />;
+
+Preview.args = {
+  'aria-label': 'Ola Nordmann',
+  color: 'accent',
+  size: 'md',
+  variant: 'circle',
+  children: '',
+};
+
+export const NoName: Story = () => <Avatar aria-label='Ola' />;
+
+export const Sizes: Story = () => (
+  <>
+    <Avatar size='xs' aria-label='x s'>
+      xs
+    </Avatar>
+    <Avatar size='xs' aria-label='' />
```

Comment:
Suggest avoiding having examples with empthy aria-labels as we always want this to be declared. Add a descriptive name maybe? `"size xs"`?